tests/cli/workflow/balsamic/conftest.py

```diff
@@ -398,23 +396,23 @@ def balsamic_store(base_store: Store, balsamic_dir: Path, helpers) -> Store:
         _store,
         internal_id="balsamic_case_tgs_paired",
         family_id="balsamic_case_tgs_paired",
+        data_analysis="BALSAMIC",
```

Comment:
Do we check input data and map it? It would be nice if we ensure that this field have a value that is in a restricted vocabulary. We use the function  `ilike(%balsamic%)` in the code, is it not better to know what is there?

Reply:
Yes! Do you think it should be within the scope of this PR?

Reply:
What do you think?

Reply:
I think it might be more suitable in the PR for splitting data_analysis into data_analysis and data_delivery